### PR TITLE
Update Util.java

### PR DESCRIPTION
**Pull request by fredchen306**
Add test

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/fredchen306/sd_doc/pull/1" target="_blank">ReviewNinja

File: Util.java [hunk 1]
```diff
@@ -57,7 +57,7 @@ public void writeFile(String filePath, String sets) throws IOException {
 	    out.write(sets);
 	    out.println();
 	    fw.close();
-	    out.close();
+	    out.close(); // test
```
Comment (fredchen306) [inline Util.java]: please fix !fix

**Comment by fredchen306** [comment]
for testing 1111

File: Util.java (hunk 1 above)
Comment (fredchen306) [inline Util.java]: !fixed

**Pull request merged**